src/drive.js

```diff
@@ -471,6 +465,17 @@ async function saveWallet (walletDataList: any, encryptedWalletFileData: any) {
     name: 'Readme.txt',
     content: ''
   })
+  
+  files = await listFiles(DRIVE_SPACE, null, true, ROOT_FOLDER_NAME)
```

Comment:
Change issue title to better reflect the problem.

Reply:
fixed